### ISSUE TITLE
[Question | Not a bug]: Website down or otherwise unreachable

### ISSUE THREAD

**Issue by Griefed** [pf4j/pf4j#506]
Hello everyone!

I'm absolutely loving pf4j and I'm using it to provide extension functionality to a passion project of mine.
During the writing of some documentation and putting in links to the pf4j docs, I noticed that the website has been unavailable for a couple of days now. Is there any ETA on when it may become available again?

I've also noticed that the sponsoring button in the pf4j-repo doesn't lead anywhere. I would love to support this project, and maybe take a couple of bucks out of the hosting costs for the pf4j website by sponsoring you on a regular basis, if possible. In what way may one provide some financial support for pf4j?

Future versions of my project will heavily rely on pf4j, as I hope to garner interest in it and potential extension-developers. Noticing the website-trouble and the missing sponsor-links, I began to worry a little.

I hope everything is fine and well on your end. 

Cheers,
Griefed

**Comment by decebals** [comment]
@Griefed 
I didn't know about this problem, thank you for notification.
I paid for the renewal of the domain but now I see that are some problems. I hope that in 1-2 days the problem will be solved (it's not about money).
Thanks for the kind words, I try to give back to the open source community what I have consumed throughout my career. I will continue to try to pay the hosting as well, as much I can.
For the moment I don't have that much time left to allocate to open source projects. Fortunately pf4j is a mature project, which only has a few bugs and those are not major/blockers (from what I know).
It's interesting/strange how life takes you sometimes, it's hard for you to find a good job even if you have tangible experience (public projects, ..). This project is 100% a community driven project, there is no company behind it, only people passionate about what they do. Over time I was lucky to have quality collaborators who pushed the project forward, it's not just my credit. I will never stop thanking these people. In the end is not my project, I was only the its initiator :smile:.
I would have loved to work on it more, but I have a family and bills to pay. 
It's hard to work at your daily job during the day and to do open source in the evening and on the weekend, at least after an age. I still have motivation, but I don't have much time. 
I would have liked to work on this project as part of my daily job, but unfortunately it is not possible, those who are already doing this are lucky.
In conclusion, you don't have to worry, I will try to support this project as much as I can. The fact that someone uses it and cares about it is enough for me to continue.

**Comment by Griefed** [comment]
Heya decebals,

I'm in a similar boat, regarding the fulltime-job, sparetime, bills and family. Luckily, I can get some time into my project(s) every now and then and still continue things as a hobby. I fully understand how you feel. Life *does* tend to do very strange things. I know my life has thrown some crazy things at me. Looking back at things, one tends to realize that not all may have been well, but things got us where we are today. And so long as one is happy with where they are today, all is well that ends well, I'd say. 😁

I hope your current job is a good one, which makes you and your family happy as well as paying the bills. I certainly wish you the best.

We're all incredibly lucky to have you around, as without you, this project and everything which revolved around it, wouldn't exist. So thanks for taking the time out of your day and responding in such length to my small questions, and thanks for the smalltalk!

I'll go ahead and close this issue then.
Take care, stay safe and stay awesome.

Cheers,
Griefed

**Comment by decebals** [comment]
The site is up again.

**Comment by Griefed** [comment]
Thank you very much for letting me know!
